src/SessionSetup/ServiceDeployManager.cpp

```diff
@@ -348,8 +351,8 @@ void ServiceDeployManager::CopyFileToLocalImpl(orbit_base::Promise<ErrorMessageO
   ORBIT_LOG("Copying remote \"%s\" to local \"%s\"", source, destination);
 
   // NOLINTNEXTLINE - Unfortunately we have to fall back to a raw `new` here.
-  auto operation =
-      new orbit_ssh_qt::SftpCopyToLocalOperation{&session_.value(), sftp_channel_.get()};
+  auto operation = new orbit_ssh_qt::SftpCopyToLocalOperation{
+      &session_.value(), sftp_channel_.get(), std::move(stop_token)};
```

Comment:
I can't make a comment in the code below, that's why it's here.

When the operation gets cancelled, the returning future will report that the operation finished successfully. I think in that case we should return an error. This could be done below by checking whether a a stop was requested and setting the result accordingly.

Reply:
done!